tests/parametric/test_otel_span_methods.py

```diff
@@ -45,10 +42,35 @@ def test_otel_start_span(self, test_agent, test_library):
         assert root_span["meta"]["start_attr_key"] == "start_attr_val"
         assert root_span["duration"] == duration * 1_000  # OTEL expects microseconds but we convert it to ns internally
 
-    @irrelevant(context.library == "cpp", reason="library does not implement OpenTelemetry")
-    @missing_feature(context.library == "php", reason="Not implemented")
-    @missing_feature(context.library == "ruby", reason="Not implemented")
-    def test_otel_set_service_name(self, test_agent, test_library):
+    @missing_feature(context.library == "golang", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "java", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "nodejs", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "dotnet", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "python", reason="New operation name mapping not yet implemented")
+    @missing_feature(context.library == "python_http", reason="New operation name mapping not yet implemented")
+    def test_otel_start_span(self, test_agent, test_library):
+        """
+            - Start/end a span with start and end options
```

Comment:
So this is really checking that the `span_kind` is used as the operation name, by default, if nothing else is provided, right? Or is it testing something else?

Reply:
So this was a pre-existing one (I also previously added an `old_name` version of this but now removed in 4cb9bb49946547e19f35485a611bea6e56a23da2).

But yes, this is checking that the `span_kind` will be used for the operation name now, but also I think this one is specifically that the duration is set correctly